src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/AzureRM.CognitiveServices.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.9.10'
+ModuleVersion = '0.9.11'
```

Comment:
@yangyuan please revert this change -- we will handle the versioning at a later time

Reply:
Done undo